spec/diddocuments.md

```diff
@@ -223,7 +223,7 @@ KERI AID public keys can be used to sign a variety of data.  This includes but i
 References to verification methods in the DID document MUST use the relative form of the identifier, e.g., `"authentication": ["#<identifier>"]`.
 
 #### Service Endpoints
-Did:webs supports service endpoint types declared in the DID Specification Registries, such as [DIDCommMessaging](https://www.w3.org/TR/did-spec-registries/#didcommmessaging).  For additional details about the mapping between KERI events and the Service Endpoints in the DID Document, see [Service Endpoint KERI events](#service-endpoint-event-details).  It is important to note that DID document service endpoints are different than the KERI service endpoints detailed in [KERI Service Endpoints as DID Document Metadata](#keri-service-endpoints-as-did-document-metadata). 
+Did:webs supports service endpoints, including types declared in the DID Specification Registries, such as [DIDCommMessaging](https://www.w3.org/TR/did-spec-registries/#didcommmessaging).  For additional details about the mapping between KERI events and the Service Endpoints in the DID Document, see [Service Endpoint KERI events](#service-endpoint-event-details).  It is important to note that DID document service endpoints are different than the KERI service endpoints detailed in [KERI Service Endpoints as DID Document Metadata](#keri-service-endpoints-as-did-document-metadata). 
```

Comment:
The `did:webs` at the start of the line should not have a capital “D”, and should probably be in back ticks.

Reply:
Good eye, fixed in https://github.com/trustoverip/tswg-did-method-webs-specification/pull/59/commits/179f3d68125617c6564f296b80a964bd5a4526d7